semantic-conventions/syntax.md

```diff
@@ -225,8 +223,8 @@ An attribute is defined by:
     * `"double[]"`: Array of double attributes.
     * `"boolean[]"`: Array of booleans attributes.
   * _template type as string literal:_ `"template[<PRIMITIVE_OR_ARRAY_TYPE>]"` (See [below](#template-type))
-
   See the [specification of Attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/README.md#attribute) for the definition of the value types.
+- `stability`, enum - either `stable` or `experimental`, specifies the stability of the attribute. Required.
```

Comment:
I would've expected that we'd need to adapt the schema (used for IDE hints) as well but it looks like we forgot to introduce `stability` there in the first place:
https://github.com/open-telemetry/build-tools/blob/ca02f82a2af66901f01028c0d2a28e871cc2b5e4/semantic-conventions/semconv.schema.json
I'll open an issue so we can add this in a follow-up.

Reply:
https://github.com/open-telemetry/build-tools/issues/276